dashboard/src/components/RpcErrorMessage/RpcErrorMessage.scss

```diff
@@ -0,0 +1,9 @@
+.error-alert-rpc {
+  ul > li {
+    font-weight: bolder;
```

Comment:
We are using explicit numbers in the rest of the CSS files, usually 400, 500 or 600. See what you think

Reply:
Sure, will change it to be aligned 😃